src/assets/v1.3.0/sign_message_lib.json

```diff
@@ -24,6 +24,7 @@
     "42161": "0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2",
     "42220": "0x98FFBBF51bb33A056B08ddf711f289936AafF717",
     "43114": "0x98FFBBF51bb33A056B08ddf711f289936AafF717",
+    "80001": "0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2",
```

Comment:
Wrong order here, but not blocking and can be fixed in a separate PR

Reply:
thanks @rmeissner, here's the PR https://github.com/gnosis/safe-deployments/pull/51